pkg/source/markdown.go

```diff
@@ -77,13 +89,21 @@ func (m *markdown) Output() string {
 		s = strings.TrimSpace(s)
 	}
 
+	if !m.OmitNotice() {
+		s = fmt.Sprintf(confluenceEditNotice, m.opts.DoxNoticeFileUrl) + s
+	}
+
 	return s
 }
 
 func (m *markdown) Ignore() bool {
 	return m.ignore
 }
 
+func (m *markdown) OmitNotice() bool {
+	return m.omitNotice
+}
+
```

Comment:
And whether `OmitNotice()` needs to be an exported function? I don't think it gets called outside of its own package, in which case, the option in the struct could be accessed directly.